src/models/Business.model.ts

```
@@ -0,0 +1,62 @@
+            }
+        ],
+        isVerified: {
+            type: Boolean,
+            default: false
+        }
+    },
+    {
+        timestamps: true,
+        toJSON: {
+            transform(doc, ret) {
+                delete ret.__v;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)